### PR TITLE
Adds samurai armor to the theatervend

### PR DESCRIPTION
**Pull request by BomberBro**
For too long we've given out content to ninjas, which are easily the gayest and weakest of all the potential warriors we could work with. This PR aims to right this wrong. You can get samurai armor (and an oni mask, thanks Ancient) from the costumevend, and if I ever make colored variants for the armor I'll remove it from the theatervend and make it a bulk cargo order, like our knight armors.

final
Thanks to DrSnips and Ancientpowerer for advice with spriting.

:cl:
 * rscadd: Adds samurai armor to the theatervend premium.

### CODE REVIEW

**Comment by BomberBro** [comment]
My current branch is way too old for this, I'm gonna update and bring it back

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -2543,8 +2543,17 @@ var/global/num_vending_terminals = 1
 		/obj/item/clothing/head/that/magic = 3,
 		/obj/item/clothing/suit/kimono = 3,
 		/obj/item/clothing/gloves/white = 3,
+<<<<<<< HEAD
```
Comment (Kurfursten) [inline code/game/machinery/vending.dm]: FIXIT

**Review by Kurfursten — APPROVED** [review]

**Comment by tuypo1** [comment]
One day we shall have all the costumes we need to put on a performance of the entire Mikado

**Comment by Tokanova** [comment]
looking forward to colored varients so i can do ronin warriors cosplay as a ninja. loved that anime.

**Comment by SonixApache** [comment]
cute sprites :+1:
more costumevend bloat :-1:
free armor :-1:

**Comment by Kurfursten** [comment]
Sonix poses good points. There is no good reason this isn't in a smartbox

However you do have it as a coin item (same as chaplain's knight armor) so I guess it's fine in terms of balance personally

**Comment by ShiftyRail** [comment]
>Sonix poses good points. There is no good reason this isn't in a smartbox

Disagree, there's the equivalent of a free riot shield (which is infinitely more useful) in the theatervend. This doesn't change much to the powercreep equation.

**Comment by Kurfursten** [comment]
Smartboxes are not lockboxes shifty... They just reduce vendor bloat

**Comment by ghost** [comment]
Time to sprite the hairstyles like I promised.

**Comment by BomberBro** [comment]
Main reasoning for the lack of smartboxes was the same reasoning with the roman armor and shields we have, which aren't in boxes. I assumed those were separated for a reason since they also have light armor, plus needing 2 coins makes it a bit harder to get actual (though weak) armor.

**Pull request merged**